libs/langchain/tests/unit_tests/embeddings/test_caching.py

```diff
@@ -27,7 +27,7 @@ def cache_embeddings() -> CacheBackedEmbeddings:
     store = InMemoryStore()
     embeddings = MockEmbeddings()
     return CacheBackedEmbeddings.from_bytes_store(
-        embeddings, store, namespace="test_namespace"
+        embeddings, store, batch_size=50, namespace="test_namespace"
```

Comment:
Would you be able to add unit tests that cover the batching logic (e.g., batch size 3?)  -- lots of folks use CacheBackedEmbeddings so we want to make sure that nothing breaks before releasing